render/detailed/connections.go

```diff
@@ -145,7 +145,7 @@ func outgoingConnectionsSummary(topologyID string, r report.Report, n report.Nod
 				if !ok {
 					continue
 				}
-				key := newConnection(n, node, port, remoteEndpointID, localAddr)
```

Comment:
Why is this change necessary? `localEndpoint.ID` will only be used in the case of internet nodes, so the id will always be the client endpoint, requesting the outbound connection. Meanwhile we're using the remote port. Shouldn't we be using the `remoteEndpoint.ID`, to match?